edn/src/edn.rustpeg

```diff
@@ -29,31 +30,60 @@ use types::Value;
 // TODO: Support discard
 
 pub nil -> Value =
-    "nil" { Value::Nil }
```

Comment:
I feel like `rust-peg` probably solves this problem generically, marking up your tokens with spans automatically.  Did you investigate this?